README.md

```diff
@@ -25,11 +25,22 @@ Successfully built 5150f0103068
 brew cask install docker # then in apps click on docker app to get in running (whale icon in top bar)
 
 # build image
-docker build -t zostera/django-ci .  
+docker build -t zostera/django-ci .
 
 # run image in container and run django-entrypoint.sh + bash command to open bash prompt
-docker run -it zostera/django-ci bash  
+docker run -it zostera/django-ci bash
 
 # stop container
 docker container stop <container-id>
 ```
+
+## Running the tests on a container:
+
+```
+# Build image:
+docker build -t zostera/django-ci .
+
+# Run tox in the container:
+# (current directory is a checkout of https://github.com/observation/observation.org)
+docker run --mount type=bind,source=${PWD},target=/github/workspace --workdir=/github/workspace -it zostera/django-ci tox -e django
```

Comment:
Het is denk ik handiger om een `docker-compose.yml` te maken die de mount die zodat je alleen `docker compose up && docker compose exec app tox -e django` hoeft te doen.

Reply:
Ja, dat zou handig zijn.

Het voordeel van deze vind ik dat het op een willekeurige repo toepasbaar is, onafhankelijk van het bestaan van een docker-compose-file in de repo.

Reply:
Goed punt.